semanticdb/metacp/src/main/scala/scala/meta/cli/metacp/Main.scala

```diff
@@ -46,7 +46,7 @@ object Main {
               case _ => None
             }
             val semanticdbRoot = Paths.get(settings.d, "META-INF", "semanticdb")
-            val className = NameTransformer.decode(fragment.name.toString)
+            val className = NameTransformer.decode(fragment.name.toString.replace("\\", "/"))
```

Comment:
Why this change? Seems ad-hoc. Is this supposed to be `PathIO.toUnix`?

Reply:
Done.